lib/node_modules/@stdlib/blas/base/sscal/lib/sscal.js

```diff
@@ -0,0 +1,87 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// VARIABLES //
+
+var M = 5;
+
+
+// MAIN //
+
+/**
+* Multiplies `x` by a scalar `alpha`.
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {number} alpha - scalar
+* @param {Float32Array} x - input array
+* @param {PositiveInteger} stride - index increment
+* @returns {Float32Array} `x`
+*
+* @example
+* var Float32Array = require( '@stdlib/array/float32' );
+*
+* var x = new Float32Array( [ -2.0, 1.0, 3.0, -5.0, 4.0, 0.0, -1.0, -3.0 ] );
+*
+* sscal( x.length, 5.0, x, 1 );
+* // x => <Float32Array>[ -10.0, 5.0, 15.0, -25.0, 20.0, 0.0, -5.0, -15.0 ]
+*/
+function sscal( N, alpha, x, stride ) {
+	var ix;
+	var i;
+	var m;
+
+	if ( N <= 0 || stride <= 0) {
+		return x;
+	}
+
+	if ( alpha === 0.0 ) {
```

Comment:
This branch should be removed.

Reply:
Planning on comparing alpha to 1.0 instead